src/server.h

```diff
@@ -116,6 +116,7 @@ struct hdr_histogram;
 #define CONFIG_MAX_HZ            500
 #define MAX_CLIENTS_PER_CLOCK_TICK 200          /* HZ is adapted based on that. */
 #define CRON_DBS_PER_CALL 16
+#define CRON_DICTS_PER_CALL 1024 /* Number of dictionaries that can be resized by cron job during one call. */
```

Comment:
This seems high, do we need to be so aggressively looking for hash tables to resize?

Reply:
I think we can piggyback on the CRON_DBS_PER_CALL value and no need to introduce a new constant. WDYT ?